source/Nuke.Common/Tools/Unity/UnityBaseSettings.cs

```diff
@@ -10,11 +10,13 @@ namespace Nuke.Common.Tools.Unity
 {
     public partial class UnityBaseSettings
     {
+        public override string ToolPath { get; internal set; } = UnityTasks.GetToolPath();
```

Comment:
why this change?

Reply:
This change was unnecessary, reverted the last commit.